include/list.h

```diff
@@ -35,6 +35,9 @@ struct list_head {
 	struct list_head *next, *prev;
 };
 
+#define LIST_HEAD(name) \
+	struct list_head name = { .next = &(name), .prev = &(name) }
+
```

Comment:
Interestingly, we do not have this macro yet ;-)

Reply:
Ah, now I remember we had this already there, and it was removed by commit 5d74cf0092b132bf224024548e46566033c84937 due to conflict with  /usr/include/sys/queue.h

Reply:
Oh, I see. What should I do?

Reply:
Maybe add UL_ prefix.